### DISCUSSION TITLE
Navigate navigation menu items with <C-n> and <C-p>

### DISCUSSION THREAD

**Discussion by FabianUntermoser** [esm7/obsidian-vimrc-support#151] [General]
When searching for a command, I would like to be able to navigate up and down the command window list with `<C-n>` and `<C-p>` respectively. However it seems that the mappings do not work, when a menu popup is showing.
This is quite annoying when finding files as well, as then I have to reach for my Up and Down keys.

I tried the following mapping:
```
map <C-n> <Down>
map <C-p> <Up>
```

Has anybody got this working or could give me a hint why this is not working?

**Comment by arsham** [comment]
Hi @FabianUntermoser , have you found and answer to this question?

**Reply by FabianUntermoser** [reply]
@arsham Currently I'm using the [QuickSwitcher++](https://github.com/darlal/obsidian-switcher-plus) as a search bar which I believe supports these mappings out of the box.

**Reply by mirql** [reply]
im looking for solution for other cases like `/` completions and all other menus
